cmd/logcli/main.go

```diff
@@ -597,6 +597,7 @@ func newQuery(instant bool, cmd *kingpin.CmdClause) *query.Query {
 	cmd.Flag("no-labels", "Do not print any labels").Default("false").BoolVar(&q.NoLabels)
 	cmd.Flag("exclude-label", "Exclude labels given the provided key during output.").StringsVar(&q.IgnoreLabelsKey)
 	cmd.Flag("include-label", "Include labels given the provided key during output.").StringsVar(&q.ShowLabelsKey)
+	cmd.Flag("include-common-labels", "Include common labels in the main output.").BoolVar(&q.IncludeCommonLabels)
```

Comment:
```suggestion
	cmd.Flag("include-common-labels", "Include common labels in output for each log line.").BoolVar(&q.IncludeCommonLabels).Default("false")
```

Reply:
Good point! Applied.

Reply:
Hm, seems like the `Default()` needs to be before `BoolVar()`, sorry.

Reply:
Fixed.